exercicios/ex3/action.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+function checkinResult($name) {
+    $sex = $_POST['genero'];
+    $age = $_POST['idade'];
+    $name = $_POST['nome'];
+
+    if ($sex === 'F' && $age < 25) {
+        $result = $name;
+    } else {
+        $result = $name;
+    }
+    return $result;
+}
```

Comment:
Não compreendi essa logica, se for feminino retorna o nome e menor de 25 e não for também.